events/models.py

```diff
@@ -200,7 +200,7 @@ class Organization(MPTTModel, BaseModel):
     parent = TreeForeignKey('self', null=True, blank=True, on_delete=models.SET_NULL,
                             related_name='children')
     admin_users = models.ManyToManyField(
-        User, blank=True, related_name='admin_organizations'
+        User, blank=True, related_name='admin_old_organizations'
```

Comment:
This is because the `Organization` model in `django_orghierarchy` has the same field with the same related_name.